ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Assertions.scala

```diff
@@ -66,20 +64,17 @@ object Assertions {
     )
   }
 
+  //KTODO: remove this two-level function
```

Comment:
```suggestion
  // TODO error codes daml 2.0: remove this two-level function
```
I'd rather you used more standard TODO markings. Applies throughout

Reply:
Right, I shouldn't have used my custom naming. Changing 👍

Reply:
d909e584ccef30fa639838c4b8e46e8606240a3f